app/src/main/AndroidManifest.xml

```diff
@@ -64,6 +64,22 @@
             android:label="@string/action_settings"
             android:parentActivityName=".PasswordStore" />
 
+        <activity
+            android:name=".crypto.PasswordCreationActivity"
+            android:label="@string/new_password_title"
+            android:parentActivityName=".PasswordStore"
+            android:windowSoftInputMode="adjustResize" />
+
+        <activity
+            android:name=".crypto.DecryptActivity"
+            android:parentActivityName=".PasswordStore"
```

Comment:
Should this also get a label given that `PasswordCreationActivity` has one?

Reply:
I've added a title assignment in DecryptActivity to give it a dynamic label instead.